exportoptions/properties.go

```diff
@@ -165,3 +165,8 @@ const InstallerSigningCertificateKey = "installerSigningCertificate"
 
 // SigningStyleKey ...
 const SigningStyleKey = "signingStyle"
+
+// DestinationKey ...
+const DestinationKey = "destination"
+
+const DestinationExport = "export"
```

Comment:
Will this be used as a default value starting from Xcode 15.4? Just asking because the other values in the file are named as `[key-name]Default = value`. So in this case it would be `DestinationDefault`.